CONTRIBUTING.md

```diff
@@ -2,13 +2,13 @@
 Thank you for taking your time to contribute to the SAP Cloud SDK!
 
 ## Project Structure
-This project containes multiple packages, that are managed using [lerna](https://github.com/lerna/lerna).  Productive packages are located in the [`packages`](./packages) directory, test packages are located in the [`test-packages`](./test-packages) directory.
-All dependencies of the packages are hoisted into the root project. Some of the packages are interdependent, therefore `npm install` won't work from within those packages. Run `npm install` in the root directory instead.
+This project containes multiple packages, that are managed using [lerna](https://github.com/lerna/lerna) and [yarn workspaces](https://classic.yarnpkg.com/en/docs/workspaces/).  Productive packages are located in the [`packages`](./packages) directory, test packages are located in the [`test-packages`](./test-packages) directory.
+All dependencies of the packages are hoisted into the root project. Some of the packages are interdependent, therefore `yarn install` won't work from within those packages. Run `yarn install` in the root directory instead.
```

Comment:
Is this true? I guess only dependencies used by more then one package are hoisted?

Reply:
No you're right - not all dependencies are moved up. I fixed it.